awesomemotive/affiliatewp-order-details-for-affiliates
# Issue/87 - Added data sanitization to shortcode args

Issue #87 

This is just to ensure that the shortcode arguments are escaped properly before going ahead.
The test is just to certify if the shortcode runs properly after these addition.

**Steps to test:**

1) Install **Order Details For Affiliates** addon;
2) Install **EDD** or **WooCommerce** and simulate an order creation, you can do that using our **AffiliateWP CLI Generator** if you prefer;
3) The idea is to create referrals for any affiliate you have in your database;
4) Go to AffiliateWP -> Settings -> Integrations and check the **Allow Global Access** option;
5) In WP Admin, go to Pages and use the shortcode `[affiliate_order_details affiliate_id="1" number="10" status="paid, unpaid"]`, change the affiliate_id parameter with the ID with your affiliate with the referrals created earlier, try to change the `number` and status `parameter` and check if is working properly;
6) You can also try using some JS injection like this `[affiliate_order_details affiliate_id="1" number="'); alert(1); ('" status="'); alert(1); ('"]`, it should not display any JS alerts.


**Housekeeping**


**Proposed changes**
